client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx

```diff
@@ -849,7 +802,6 @@ const UnifiedDesignPickerStep: Step = ( { navigation, flow, stepName } ) => {
 					screenshot={ fullLengthScreenshot }
 					isExternallyManaged={ selectedDesign.is_externally_managed }
 					isVirtual={ selectedDesign.is_virtual }
-					disableGlobalStyles={ disableCheckoutImmediately }
```

Comment:
This is the only reference where `disableGlobalStyles` is set, since `disableCheckoutImmediately` is essentially a check for the design-first flow, we can deprecate the `disableGlobalStyles` prop.